target/arduino/export.bat

```diff
@@ -0,0 +1,89 @@
+@ECHO off > nul
+:: Batch file to install BasicMAC from the git repository in to an
+:: existing Arduino installation by specifying the location of the 
+:: User library directory e.g.: "C:\Users\UserName\Documents\Arduino\libraries"
+
+:: \name Remko Welling (pe1mew@gmail.com)
+:: \date 8-7-2020
+:: \version 1.0
```

Comment:
I'd rather not have a date and version in here, those are likely to get outdated and git can already track this better. Having your name as author here is fine, of course.

Reply:
It's normal behaviour for me to try to keep as much compatible with Doxygen documentation. There is no need to keep versioning here. I'll remove it.